packages/frontend/src/pages/SavedDashboards.tsx

```diff
@@ -28,13 +27,6 @@ const SavedDashboards = () => {
     const [isCreateDashboardOpen, setIsCreateDashboardOpen] =
         useState<boolean>(false);
 
-    const {
-        isLoading: isCreatingDashboard,
-        isSuccess: hasCreatedDashboard,
-        mutate: createDashboard,
-        data: newDashboard,
-    } = useCreateMutation(projectUuid);
-
```

Comment:
hmm is this intended? don't we need this?

Reply:
will wait for confirmation and then approve this!

Reply:
yep, check out `packages/frontend/src/components/common/modal/DashboardCreateModal.tsx`